src/rma/destruction/admin.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib import admin
+
+from fsm_admin.mixins import FSMTransitionMixin
+
+from .models import (
+    DestructionList,
+    DestructionListItem,
+    DestructionListItemReview,
+    DestructionListReview,
+)
+
+
+class DestructionListItemInline(admin.TabularInline):
+    model = DestructionListItem
+    fields = ("zaak", "status")
+    readonly_fields = ("status",)
+    extra = 1
+
+
+class DestructionListReviewInline(admin.TabularInline):
+    model = DestructionListReview
+    fields = ("author", "text", "status")
+    extra = 1
+
+
+@admin.register(DestructionList)
+class DestructionListAdmin(FSMTransitionMixin, admin.ModelAdmin):
```

Comment:
Please make the `author` and `assignee` `raw_id_fields`

Reply:
`raw_id_fields` are added